.github/workflows/windows.yml

```diff
@@ -223,6 +220,12 @@ jobs:
         run: |
           "$env:WIX\bin" >> $env:GITHUB_PATH
 
+      # Always install crown, even on self-hosted runners, because it is tightly
+      # coupled to the rustc version, and we may have the wrong version if the
+      # commit we are building uses a different rustc version.
+      - name: Install crown
+        run: cargo install --path support/crown --force
```

Comment:
Why `--force`, when we do not need to replace binary if it's not updated?

Reply:
`--path` always behaves as if `--force` is passed, i was just making it explicit :)